browser/src/types.d.ts

```diff
@@ -74,3 +177,178 @@ export type Transcript = {
 }
 
 export type Strand = '+' | '-'
+
+export type ClinvarVariant = {
```

Comment:
Another case where there are many things here not obviously fitting under the title of this PR that would be better as it's own smaller PR.